connection_producer.go

```diff
@@ -2,11 +2,12 @@ package mongodbatlas
 
 import (
 	"context"
+	"github.com/hashicorp/vault/sdk/helper/useragent"
 	"sync"
 
 	"github.com/Sectorbob/mlab-ns2/gae/ns/digest"
 	"github.com/hashicorp/vault/sdk/database/helper/connutil"
-	"github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	"go.mongodb.org/atlas/mongodbatlas"
```

Comment:
Was this change intended to support scopes? Can our client be extended if so?

Reply:
Ah I should have explained in the description, this is actually the same module, but they have updated the package path since the version we were on before: https://github.com/mongodb/go-client-mongodb-atlas/pull/87